crates/cairo-coverage/tests/data/coverage_ignore/Scarb.toml

```diff
@@ -3,18 +3,11 @@ name = "coverage_ignore"
 version = "0.1.0"
 edition = "2024_07"
 
-# See more keys and their definitions at https://docs.swmansion.com/scarb/docs/reference/manifest.html
 [dependencies]
 starknet = ">=2.8.0"
 
 [dev-dependencies]
-snforge_std = "0.30.0"
```

Comment:
Change to to git because of this https://github.com/software-mansion/cairo-coverage/pull/111#issuecomment-2523387203